### ISSUE TITLE
A11y: Non-interactive element <label> cannot have interactive role -> for 'option', 'radio' and 'tab'

### ISSUE THREAD

**Issue by shadowempire123** [skeletonlabs/skeleton#1191]
### Current Behavior

"@skeletonlabs/skeleton": "^1.0.0",
...

 VITE v4.2.1  ready in 1119 ms

  ➜  Local:   http://localhost:8082/
  ➜  Network: use --host to expose
  ➜  press h to show help
15:42:05 [vite-plugin-svelte] ssr compile in progress ...
15:42:06 [vite-plugin-svelte] /node_modules/@skeletonlabs/skeleton/components/ListBox/ListBoxItem.svelte:54:0 A11y: Non-interactive element <label> cannot have interactive role 'option'
15:42:06 [vite-plugin-svelte] /node_modules/@skeletonlabs/skeleton/components/Radio/RadioItem.svelte:37:0 A11y: Non-interactive element <label> cannot have interactive role 'radio'
15:42:06 [vite-plugin-svelte] /node_modules/@skeletonlabs/skeleton/components/Tab/Tab.svelte:36:0 A11y: Non-interactive element <label> cannot have interactive role 'tab'

**Comment by endigo9740** [comment]
@shadowempire123 we're aware of SvelteKit's recent addition of new A11y warnings and we're working to address those alongside updates to the packaging process in recent updates here:
https://github.com/skeletonlabs/skeleton/pull/1170
